demo/performance/react.html

```diff
@@ -1,7 +1,12 @@
 <!doctype html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="utf-8">
+    <!-- TODO: The react library has a bunch of dependencies which makes the CSP
```

Comment:
Where we were forces to loosen our policy _more_ than we’d like… I added a `TODO` comment. Here, for example, it’s brutal to try and enumerate all the individual `esm.sh` urls for transitive dependencies of React. So, I just allow any package there via `https://esm.sh/`. This is just a demo, so I’m not terribly worried about it, but I think it’s good to call out when integration is cumbersome.

By contrast, the `lit-html` policy was actually a breeze as it doesn’t _have_ dependencies (at least not for our usage in the demo).